LocalWiki.php

```diff
@@ -27,6 +27,39 @@ function onBeforePageDisplay( OutputPage $out ) {
 			$out->addMeta( 'og:image:width', '1200' );
 		}
 
+		break;
+	case 'gratisdatawiki':
+		$wgHooks['overridePageMetaTags'][] = 'overridePageMetaTags';
+		$wgHooks['BeforePageDisplay'][] = 'onBeforePageDisplay';
+		// The title of wikibase entities will be the label, if available, or else the entity id (e.g. 'Q42').
+		function overridePageMetaTags( OutputPage $outputPage ) {
```

Comment:
Please call onX as standard for hooks

Reply:
> Please call onX as standard for hooks

I was almost asking a question about that, I thought it doesn't matter...

Reply:
@RhinosF1 like this?

Reply:
each word excluding the first should be a capital so onOverridePageMetaTags

Reply:
> each word excluding the first should be a capital so onOverridePageMetaTags

Okay 👍

Reply:
@RhinosF1 we good with this?

Reply:
Nope, see comment below